fileHub/js-app/app/user-page/file-list.js

```diff
@@ -0,0 +1,56 @@
+import {Component} from '../components/component.js';
+import {FileListItemView} from './file-list-item-view.js';
+
+/**
+ * Component that renders folder content list.
+ */
+export class FileList extends Component {
+  /** @inheritDoc */
+  _initNestedComponents() {
+    if (this._fileItems && this._fileItems.length > 0) {
+      const tableElement = this._getElement('fileListItems');
```

Comment:
Please create constant for `fileListItems` and use it both in _initNestedComponents and markup.